certbot-route53/setup.py

```diff
@@ -3,23 +3,28 @@
 from distutils.core import setup
 from setuptools import find_packages
 
-version = '0.1.5'
+version = '0.15.0.dev0'
 
 install_requires = [
-    'acme>=0.9.0',
-    'certbot>=0.9.0',
-    'zope.interface',
+    'acme=={0}'.format(version),
+    'certbot=={0}'.format(version),
     'boto3',
+    'mock',
+    # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
+    # will tolerate; see #2599:
+    'setuptools>=1.0',
+    'zope.interface',
 ]
 
 setup(
     name='certbot-route53',
     version=version,
-    description="Route53 plugin for certbot",
+    description="Route53 DNS Authenticator plugin for Certbot",
     url='https://github.com/lifeonmarspt/certbot-route53',
```

Comment:
I think this line should be deleted. I'm assuming this was an accident, but if you did it on purpose to try and give a nod to the original maintainers, I think we could do that in a different way.